### PR TITLE
include gains workshop initial page

### PR DESCRIPTION
**Pull request by robert4gains**
GAINS workshop to index
GAINS workshop initial page

### CODE REVIEW

File: gains_workshops.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
+GAINS Workshops
+^^^^^^^^^^^^^^^^^^
```
Comment (danielhuppmann) [inline gains_workshops.rst]: Not strictly necessary, but better to keep the title and the syntax-highlighting in sync
```suggestion
^^^^^^^^^^^^^^^
```

File: gains_workshops.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+use cases of some features are shown. The user can learn how
+to build their own emission scenarios, adjust and create new policy
+constraints and run analysis on costs and co - benefit.
+For information about the model, its structure, mathematical
+formulation and much more, please see the `documentation <http://gains.iiasa.ac.at/models/gains_resources.html>`_.
+
+.. contents:: Table of Content
+   :local:
+
+GAINS Capacity building
```
Comment (danielhuppmann) [inline gains_workshops.rst]: Not quite what this line does? Is this meant as a heading? If yes, please add syntax highlighting

File: gains_workshops.rst [hunk 3]
```diff
@@ -0,0 +1,24 @@
+GAINS Workshops
+^^^^^^^^^^^^^^^^^^
+
+The aim of the workshop is to help new users of the GAINS
+modelling framework getting started with their modelling work.
+The main features of the “framework” are introduced, and the
+use cases of some features are shown. The user can learn how
+to build their own emission scenarios, adjust and create new policy
+constraints and run analysis on costs and co - benefit.
```
Comment (danielhuppmann) [inline gains_workshops.rst]: Minor editorial suggestion
```suggestion
constraints and run analysis on costs and co-benefits.
```

File: index.rst [hunk 4]
```diff
@@ -40,8 +40,12 @@ learned directly - and of course to rewind as often as you like and need.
    :caption: Workshops and tutorials
 
    message_ix_workshop
+   gains_workshops
 
 - :doc:`message_ix_workshop`
+- :doc:`gains_workshops`
+
+
```
Comment (danielhuppmann) [inline index.rst]: No need to add extra lines here...

**Review by danielhuppmann — CHANGES_REQUESTED** [review]
Thanks @robert4gains - a few suggestions inline, then good to go!

**Comment by robert4gains** [comment]
All recommended changes done

**Review by danielhuppmann — APPROVED** [review]
Looks good to me as a first step towards including GAINS in the ECE teaching pages!
Over to @LauWien for a final review and merge.

File: index.rst [hunk 5]
```diff
@@ -40,8 +40,12 @@ learned directly - and of course to rewind as often as you like and need.
    :caption: Workshops and tutorials
 
    message_ix_workshop
+   gains_workshops
 
 - :doc:`message_ix_workshop`
+- :doc:`gains_workshops`
+
```
Comment (LauWien) [inline index.rst:47]:
```suggestion
```

File: index.rst (hunk 4 above)
Comment (LauWien) [inline index.rst]:
```suggestion
```

**Review by LauWien — CHANGES_REQUESTED** [review]
Thank you! Looks good to me. I would just delete the two additional lines 47 and 48 in the _index.rst_ file as suggested.

**Review by LauWien — APPROVED** [review]

**Pull request merged**